Packs/Legacy/Playbooks/playbook-BPA.yml

```diff
@@ -47,6 +47,9 @@ tasks:
     nexttasks:
       '#none#':
       - "3"
+    scriptarguments:
+      generate_zip_bundle:
+        simple: "true"
```

Comment:
```suggestion
        simple: "false"
```

Reply:
yes, set the parameter to false as default